.github/workflows/androidRelease.yml

```
@@ -0,0 +1,83 @@
+#This is a workflow that creates, downloads, uploads and releases APK-file.
+name: APK Workflow
+
+#Event triggers
+#execute the jobs everytime when a pull_request to the main-branch is created
+on:
+  pull_request:
```

Wouldn't it be better to create a release when you push on the main branch instead of when a PR comes.
```suggestion
  push:
```